backend/app/api/endpoints/dingtalk_wikispace.py

```
@@ -0,0 +1,128 @@
+    db: Session = Depends(get_db),
+    current_user: User = Depends(get_current_user),
+) -> DingtalkSyncStatus:
+    """Get the sync status for the current user's DingTalk wikispace nodes."""
+    status = DingTalkWikiSpaceService.get_sync_status(current_user, db)
+    return DingtalkSyncStatus(**status)
+
+
+@router.delete("/{node_id}")
+def delete_synced_wikispace_node(
+    node_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)